be/src/exec/hdfs_scanner_text.cpp

```diff
@@ -280,19 +282,16 @@ Status HdfsTextScanner::parse_csv(int chunk_size, ChunkPtr* chunk) {
             const auto& slot = _scanner_params.materialize_slots[j];
             DCHECK(slot != nullptr);
 
-            const std::string formatted_slot_name =
-                    _scanner_params.case_sensitive ? slot->col_name() : boost::to_lower_copy(slot->col_name());
-
             size_t chunk_index = _scanner_params.materialize_index_in_chunk[j];
-            size_t csv_index = _formatted_hive_column_name_2_index[formatted_slot_name];
+            size_t csv_index = _materialize_slots_index_2_csv_column_index[j];
             Column* column = _column_raw_ptrs[chunk_index];
             if (csv_index < fields.size()) {
                 const Slice& field = fields[csv_index];
                 options.type_desc = &(_scanner_params.materialize_slots[j]->type());
                 if (!_converters[j]->read_string(column, field, options)) {
```

Comment:
why not reorder `_converters` to make it as same order as `csv_index`

Reply:
good idea